### DISCUSSION TITLE
nmrXiv recommendations for NMR MiChI

### DISCUSSION THREAD

**Discussion by NRayya** [NFDI4Chem/nmrxiv#830] [Minimum Information Standards]
For the [NMR Task Force Meeting taking place in Jena](https://www.nfdi4chem.de/index.php/event/nmr-task-force-meeting/), nmrXiv will recommend a bundle of minimum requirements for NMR data (synthetic compounds) including the sample preparation and the assay setup and results. The recommendations come from previous discussions with Chemotion team to provide NMR templates in the ELN containing the minimum information. The resulted template was further proposed to the rest of NFDI4Chem in a TA2 monthly meeting and received positive feedback. 

More discussion is needed to decide if this is the minimum or more than needed. If the recommended MIChI receive positive feedback in the workshop too, it will be adopted by nmrXiv and published on the NFDI4Chem website for MiChI workshops as a first version, with the requested details described using ontologies.

The following image shows our recommendations for synthetic sample metadata:

![image](https://github.com/NFDI4Chem/nmrxiv/assets/82588017/55c57b99-c181-4f75-8a67-767226c9f810)

And here you can find NMR assay metadata (we have to add the nucleus here):
![image](https://github.com/NFDI4Chem/nmrxiv/assets/82588017/7fb3081f-d886-4c5d-be11-91ca0ca067d7)

In addition to NMR metadata, we will additionally highlight the needed data files (raw files, strucutre files..), and the use of formats readable by nmrXiv and including a set of results. 

All comments are welcome

**Comment by NRayya** [comment]
Related work: https://docs.google.com/document/d/1NAR91f3UzikR3PbgA2ZJ0zOJTphv8yH0HFWGRV90lOw/edit#heading=h.igdcuyo01x7l

**Comment by stefhk3** [comment]
Zwei Fragen:
1. Was ist pulse sequence? Der name des pulse programs? Die sind doch eigentlich willkürlich?
2. Impliziert software used for data processing, dass die processed data enthalten sein müssen? Wenn nein, ist die Angabe dann sinnvoll? Wenn ja, vermischt das nicht die Ebenen?

**Reply by NRayya** [reply]
Good evening Stefan,

We currently have a more elaborate draft for our recommendations, which you can find here: https://docs.google.com/document/d/15fSV7bkeULqHSwkMjqosecnVydv5Smp0pw9R6NStPxg/edit#heading=h.33wpiyywo9ez 

It provides definitions of the requested metadata

**Comment by NRayya** [comment]
To be closed in favor of the available NMR MIChI now. Further discussions in the NMR task force meeting
